src/main/java/fi/aalto/cs/apluscourses/presentation/exercise/ExerciseViewModel.java

```diff
@@ -36,9 +36,11 @@ public String getPresentableName() {
    * Returns {@code true} if the exercise is submittable from the plugin, {@code false} otherwise.
    */
   public boolean isSubmittable() {
-    String presentableName = getPresentableName();
-    return presentableName.length() > "Assignment xx (".length()
-        && !"Assignment 1 (Piazza)".equals(presentableName);
+    // O1_SPECIFIC
+    String name = getPresentableName();
+    int maxSubmissions = getModel().getMaxSubmissions();
+    return name.length() > "Assignment xx (".length() && !"Assignment 1 (Piazza)".equals(name)
```

Comment:
can this be a method that we would "all know about but pretend it does not exist and never open it again"?

Reply:
the wider question: do we need this to be a flag on A+ side instead of all this... ?

Reply:
Yes I agree :smile: as far as we know, this method doesn't exist